modules/preview-react/loading-sparkles/lib/LoadingSparkles.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {keyframes} from '@emotion/react';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {canvas} from '@workday/canvas-kit-react/tokens';
+import {styled} from '@workday/canvas-kit-react/common';
+import {sparkleIcon} from './sparkleIcon';
+
+/**
+ * Duration of the sparkle animation (in ms).
+ */
+const ANIMATION_DURATION_MS = 1230;
+
+// TODO: Replace with the actual color tokens when they are available.
+/**
+ * Dragon Fruit colors.
+ */
+const AI_COLORS = {
+  dragonFruit100: '#FBF1FF',
+  dragonFruit200: '#EFD3FF',
+  dragonFruit300: '#BE61F6',
+  dragonFruit400: '#8C17D2',
+  dragonFruit500: '#6B11A3',
+  dragonFruit600: '#4A0D71',
+};
+
+/**
+ * The animation for the sparkle.
+ */
+const LOADING_ANIMATION = keyframes({
+  '0%, 79%, 100%': {
+    opacity: 0.2,
+    transform: 'scale(0.55)',
+  },
+  '27%': {
+    opacity: 1,
+    transform: 'scale(1)',
+  },
+  '53%': {
+    opacity: 0.6,
+    transform: 'scale(0.7)',
+  },
+});
```

Comment:
I shifted the animation a bit, but here is some detail on the animation grabbed from the Figma:
- 330ms: small to large (ease-in-out)
- 320ms: large to medium (ease-in)
- 320ms: medium to small (ease-in)
- 260ms: small (delay)

This translates to a 1230ms animation time with roughly 0%, 27%, 53%, 79%, 100% keyframe points.

I set the animation to `ease-in-out` but not sure if there's a better way to handle this 🤷‍♂️